instances/widgets.treasury-factory.near/widget/components/Navbar.jsx

```diff
@@ -15,6 +15,30 @@ const [showMenu, setShowMenu] = useState(false);
 const { href: linkHref } = VM.require("${REPL_DEVHUB}/widget/core.lib.url") || {
   href: () => {},
 };
+const [navbarWithAssetExchange, setNavbarWithAssetExchange] = useState([]);
+
+// we can't update all existing instances to have asset exchange so adding it here for all
+useEffect(() => {
```

Comment:
If all instances should have this by default, would it be better that the NavBar links array is not in config, and instead we only provide a map in config of the navbar items that should be disabled for the instance? I feel this way of adding a default navbar link is quite hacky, and would not be good if we need to add more.

Reply:
We don't have any more 'features' planned that we will add in navbar for now, and I was hoping we will utilise the update functionality that you did to `update_widgets` going forward, that way we can expect user to upgrade more frequently if they want to utilise new functionality, and then we can remove this.